ale_linters/erlang/erlc.erl

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env escript
```

Comment:
Sorry, I should’ve explained this earlier.

Erlang’s ecosystem doesn’t have a linker and each project has to implement it from scratch for its own needs (e.g. [1]). Erlc [2] is a compiler for Erlang, it knows nothing about build tooling that almost everyone uses (rebar3 [3][4]). You could think of it as gcc vs. cmake. It’s not possible to run rebar3 as a linter. So the script specifies paths for libraries and other compiler options.

[1] https://github.com/vim-erlang/vim-erlang-compiler/blob/master/compiler/erlang_check.erl
[2] http://erlang.org/doc/man/erlc.html
[3] http://www.rebar3.org
[4] https://github.com/erlang/rebar3